CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Fix spacing display of Transformation fields on mapping profile view page. UIDEXP-129.
 * Add job profiles deleting. UIDEXP-111.
 * Refactor to miragejs from bigtest/mirage. UIDEXP-137.
+* Implement add mapping profile transformations modal. UIDEXP-104.
```

Comment:
I think there is a visual difference between the provided mockups and your solution. According to https://issues.folio.org/secure/attachment/29338/29338_screenshot-3.png there is not a space after the transformation input field, and the name and the transformation fields look centered vertically. @VictorSoroka1 what do you think about it?

Reply:
Good catch, about centering, I noticed it back when I added transformations form initially and agree that it should be handled now. Will do.

Reply:
Agree 100% percent:)

Reply:
Please, update a screenshot after this comment fixing

Reply:
Done.

Reply:
Thanks. It looks cool now

Reply:
Thank you for catching this.